src/Pages/SignInPage.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import SignIn from '../components/SignIn/SignIn';
+
+export default function SignInPage() {
+  return (
+    <div className='bg-background bg-signin-background md:bg-contain bg-cover bg-no-repeat h-screen w-full overflow-y-auto'>
+      <SignIn />
```

Comment:
sign in and sign-up is a page, not components, so creating sign-in and sign-up components and adding it inside the sign-in page is not a good practice, in my opinion, you can put the content inside the sign-in component into the page and delete the component

Reply:
I'm not sure about that, I did it that way bc I thought it is the preferred way.

Reply:
let's wait for kutay's opinion he knows better

Reply:
Oh and writing tests for them too 😆